lib/orbit-common/cache.js

```diff
@@ -51,10 +51,9 @@ var Cache = Class.extend({
     Evented.extend(this);
 
     this.schema = schema;
-    for (var model in schema.models) {
-      if (schema.models.hasOwnProperty(model)) {
-        this._registerModel(model);
-      }
+
+    for (var i = 0, models = Object.keys(schema.models), len = models.length; i < len; ++i) {
```

Comment:
nit: I don't think the loop is very readable like this, I think the following is much easier for future developers to parse:

```js
var models = Object.keys(schema.models);
var length = models.length;

for (var index=0; index<length; index++) {
  this._registerModel(models[index]);
}
```

Also, since the inside of this loop is a function call, I'm not sure this saves any actual time...

```js
Object.keys(schema.models).forEach(this._registerModel, this);
```

Seems super easy, readable, and probably a negligible difference in terms of actual in-app performance (the cost of executing the register model function far outweighs the small gain by using a for loop here).  Also, it's unlikely that this will ever be a hot path, as it's model registration, and you are unlikely to have a ton of models anyway.  The code as written, (or using `forEach` on the object.keys i showed above) is much more readable than initializing three vars in a for loop IMO.

I think this change is unnecessary